.napari/config.yml

```
@@ -0,0 +1,16 @@
+
+# Add labels from the EDAM Bioimaging ontology
+labels:
+  ontology: EDAM-BIOIMAGING:alpha06
+  terms:
+  - 2D image
+  - 3D image
+  - Cell segmentation
+  - Connected-component analysis
+  - Illumination correction
+  - Image correction
+  - Image denoising
+  - Image thresholding
+  - Pixel classification
+  - Object classification
```

```suggestion
  - Image segmentation
  - Object feature extraction
  - Image processing
  - Smoothing
  - Morphological operation
  - Image feature detection
  - Image projection
  - Rotation
  - Spot detection
  - Semi-automatic segmentation
  - Overlap analysis
  - Edge detection
  - Top-hat transform
  - Image convolution
  - Affine transformation
  - Image crop
  - Image analysis
  - Closing
  - Dilation
  - Opening
  - Erosion
  - Linear transformation
  - Geometrical transformation
```
... I'm just adding some more relevant categories from https://github.com/edamontology/edam-bioimaging/blob/alpha06/releases/EDAM-bioimaging_alpha06.tsv